cmd/create/cluster/cmd.go

```diff
@@ -2443,6 +2530,10 @@ func run(cmd *cobra.Command, _ []string) {
 		if interactive.Enabled() {
 			command := buildCommand(clusterConfig, operatorRolesPrefix, expectedOperatorRolePath,
 				isAvailabilityZonesSet || selectAvailabilityZones, labels)
+			if err != nil {
```

Comment:
Where are we getting this error from?

Reply:
My bad, I believe I had changed the signature of `buildCommand()` and reverted it. This is a leftover. Will remove it. Thanks.